project/src/index.tsx

```diff
@@ -2,8 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app/app';
 
+type Settings = {
+  placesCount: number;
+}
+
+const Setting: Settings = {
+  placesCount: 111,
+};
+
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <App
+      placesCount = { Setting.placesCount }
```

Comment:
плохая практика так писать

```suggestion
      placesCount={Setting.placesCount}
```